server/controllers/memories-controller.js

```diff
@@ -1,27 +1,133 @@
 const {
     v4: uuidv4
 } = require('uuid')
-const models = require('../models')
+const models = require('../models');
 
 const getMemories = async (req, res) => {
-    const memories = await models.Memory.findAll();
-    if(memories.length !== 0) {
-        return res.status(200).send(memories)
+    const memories = await models.Memory.findAll({
+        raw: true
+    });
+    if (memories.length !== 0) {
+        const posts = [...memories]
+        for (let i = 0; i < posts.length; i++) {
```

Comment:
So yea, dont do this. This is exactly what "include" is for. 
If you properly associate memories and comments, you should automatically get all the comments for a memory with "include"

Reply:
If I use include I can't get the commenter's name as that comes from the user model, so I still have to get the user based on the comment's user_uuid.

Reply:
I can use include to get the comment text, but in order to get the user who posted the comment's display_name I need the comment's user_uuid, which I don't think I can get from the memory and comment association but I could be wrong!